src/functions/calcTotalPoints.jsx

```diff
@@ -6,12 +6,13 @@ function CalcPoints () {
   // console.log("calc points called");
   // console.log(JSON.stringify(allUserScores));
   const userScores = useMemo(() => {
+    if (!(Array.isArray(allUserScores) && currentUser)) return [];
```

Comment:
just a bit of guard because I faced quite some console errors on `allUserScores.filter`